BankAccountApp/BankAccountApp/BankAccount.cs

```diff
@@ -0,0 +1,88 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace BankAccountApp
+{
+    class BankAccount : GeneralAccount, IAccountOperations
+    {
+        public enum ACCOUNT_STATUS { OPEN, CLOSED };
```

Comment:
Namingul cu upper-case in stilul ACCOUNT_STATUS, era la un moment dat in guideline-uri pentru constante.
pentru enums, folosim simplu AccountStatus, iar valorile definite pentru el : Open, Closed

Reply:
De acord. Fac modificarile.